components/src/core/components/Dialog/_variables.scss

```diff
@@ -18,7 +18,7 @@
 $oxd-overlay-background-color: rgba($oxd-black-color, 0.5) !default;
 
 $oxd-dialog-close-button-font-color: $oxd-white-color !default;
-$oxd-dialog-close-button-bg-color: $oxd-interface-gray-lighten-1-color !default;
+$oxd-dialog-close-button-bg-color: #b8bdc7 !default;
```

Comment:
@KavindaK7 please check this color

Reply:
defined in https://oxd-prod-infinity.orangehrm.com/main-catalog/modals.html and in the jira ticket

![image](https://user-images.githubusercontent.com/12880059/173051726-735af492-21c1-412f-9a40-5bbfa5495333.png)